plotmanager/library/utilities/notifications.py

```diff
@@ -20,7 +22,12 @@ def _send_notifications(title, body, settings):
 
     if settings.get('notify_ifttt') is True:
         import requests
-        requests.post(settings.get('ifttt_webhook_url'), data={'value1': title, 'value2': body})
+        try:
+            requests.post(settings.get('ifttt_webhook_url'), data={'value1': title, 'value2': body}, timeout=10)
+        except requests.exceptions.Timeout:
+            logging.error(f"Connecting to IFTTT timeout.(Default is 10 sec)")
```

Comment:
Timeout doesn't necessarily mean Connection timeout as request doc mentions it catches both  
 `requests.exceptions.ConnectTimeout` , `requests.exceptions.ReadTimeout` errors.